### PR TITLE
Feat/exemplar units - LESQ-250

### PR DESCRIPTION
**Pull request by JBR90**
## Description

- Adds new Exemplar Unit component

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1861--oak-web-application.netlify.thenational.academy
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

**Review by jack-skerrett-oak — CHANGES_REQUESTED** [review]
- Could we change the name of this to "early release"? I noticed in Storybook it is under "exemplar"
- We will want some kind of tablet view to shunt things onto new lines earlier
Screenshot 2023-09-04 at 11 10 31
- The blue is different for the subject icon background on Figma
- We seem to lose the margin down the sides at mobile widths
- Button at the bottom should have "View **new**.."

**Review by jack-skerrett-oak — APPROVED** [review]
Let's get it on a page!

File: src/components/EarlyReleaseExemplarUnits/EarlyReleaseExemplarUnits.test.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { screen } from "@testing-library/react";
+
+import renderWithProviders from "../../__tests__/__helpers__/renderWithProviders";
```
Comment (kimon-satan) [inline src/components/EarlyReleaseExemplarUnits/EarlyReleaseExemplarUnits.test.tsx:3]: can you use @ paths and replace relative paths wherever you see them.

File: src/components/EarlyReleaseExemplarUnits/EarlyReleaseExemplarUnits.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import React, { FC } from "react";
+
+import Flex from "../Flex";
```
Comment (kimon-satan) [inline src/components/EarlyReleaseExemplarUnits/EarlyReleaseExemplarUnits.tsx:3]: can you use @ paths and replace relative paths wherever you see them.

File: src/components/UnitAndLessonLists/UnitList/UnitListItem/UnitListItem.tsx [hunk 3]
```diff
@@ -132,7 +145,7 @@ const UnitListItem: FC<UnitListItemProps> = (props) => {
         $gap={[8]}
         $pv={[8, 12]}
       >
-        {!isUnitOption && yearTitle && (
+        {!isUnitOption && yearTitle && !isExemplarUnit && (
```
Comment (kimon-satan) [inline src/components/UnitAndLessonLists/UnitList/UnitListItem/UnitListItem.tsx:148]: I'm confused as to why you are using a Paragraph tag which is then unstyled. This looks like it should be a heading tag ?

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units.tsx [hunk 4]
```diff
@@ -132,6 +134,7 @@ const UnitListingPage: NextPage<UnitListingPageProps> = ({
         subjectIconBackgroundColor={"lavender"}
         title={`${subjectTitle} ${examBoardTitle ? examBoardTitle : ""}`}
         programmeFactor={keyStageTitle}
+        isNew={viewType === "teachers-2023"}
```
Comment (kimon-satan) [inline src/pages/beta/[viewType]/programmes/[programmeSlug]/units.tsx:137]: Why is this needed for this PR ?

**Review by kimon-satan — APPROVED** [review]

File: src/components/UnitAndLessonLists/UnitList/UnitListItem/UnitListItem.tsx (hunk 3 above)
Comment (JBR90) [reply src/components/UnitAndLessonLists/UnitList/UnitListItem/UnitListItem.tsx:148]: I only added the '!isExemplarUnit' prop here - but yes can change to Heading makes sense.

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units.tsx (hunk 4 above)
Comment (JBR90) [reply src/pages/beta/[viewType]/programmes/[programmeSlug]/units.tsx:137]: I made some changes to the tag promo component and noticed that on the lesson and units page it was missing for new lessons/units subject icon

**Pull request merged**